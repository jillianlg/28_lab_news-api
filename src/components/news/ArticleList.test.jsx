import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Articles from './ArticlesList';

describe('ArticlesList component', () => {
  afterEach(() => cleanup());
  it('renders ArticlesList', () => {
    const { asFragment } = render(
      <Articles
        articles={[]}
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});