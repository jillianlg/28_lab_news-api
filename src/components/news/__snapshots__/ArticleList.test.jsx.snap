// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlesList component renders ArticlesList 1`] = `
<DocumentFragment>
  <h1>
    News Articles
  </h1>
  <ul
    data-testid="article"
  />
</DocumentFragment>
`;
